.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,52 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: "Chromatic"
+
+# Event for the workflow
+on: push
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Job steps
+    steps:
+      - name: Get Yarn cache path
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Checkout
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+
+      - name: Enable node
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+
+      - name: Load Yarn cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile && yarn bootstrap
+
+      - name: Build packages
+        run: yarn build
```

Comment:
My knowledge on Chakra's tooling is limited. Can you check if we need to run `build` again because afaik storybook also runs the build command for stories

Reply:
@burhanuday Yes it does, but this is monorepo managed by manypk, so all storybook dependencies has to be built first. 
For example, Button stories depends on `@chakra-ui/styled-system` which is built and managed by manypk, not storybook. 
I guess there is a way to setup webpack alias for all packages and tell Storybook to handle the transpilation and bundling of the `.ts` source files... but I'm not sure that's a good idea... there might be some differences.

This is how I understand the current setup, maybe I'm wrong. @segunadebayo can you confirm this?